src/Symfony/Component/VarDumper/Resources/functions/dump.php

```diff
@@ -14,20 +14,19 @@
 if (!function_exists('dump')) {
     /**
      * @author Nicolas Grekas <p@tchwork.com>
+     * @author Alexandre Daubois <alex.daubois@gmail.com>
      */
-    function dump(mixed $var, mixed ...$moreVars): mixed
+    function dump(mixed ...$vars): mixed
```

Comment:
If we keep the same signature, we're not able to name the first argument. The following code doesn't work:

```php
dump(first: 'Hello');
```

PHP will tell that `$var` parameter is missing. But we may throw a `\LogicException` if no argument is passed at all. Would that suit?

Reply:
no exception please, not for a debugging tool
`dump()` could just output the `^`?

Reply:
`^ 🐛`

Reply:
It seems (unfortunately) tricky to me to display only the caret (or anything else "special"), because `VarDumper:dump()` must absolutely take an argument. Obviously we can't pass `null` to it.

Could dumping `null` when no argument is passed to `dump()` be acceptable?

Reply:
We can pass a stub object, and with a custom stubcaster, display it as we want.

Reply:
@alexandre-daubois I would not dump `null`, as that's a valid dump value as well